src/features/clusters/upsert/index.tsx

```diff
@@ -112,7 +112,7 @@ export function UpsertCluster() {
 		return {
 			autoRenew: cluster?.plans?.[0]?.autoRenew ?? true,
 			systemName: cluster?.name ?? '',
-			abbreviatedName: cluster?.abbreviatedName ?? '',
+			abbreviatedName: cluster?.abbreviatedName || '',
```

Comment:
Might benefit from a comment as to why this needs `||` when the other similar lines around all use `??`?